faunadb-java-dsl/src/main/java/com/faunadb/client/types/Codec.java

```diff
@@ -45,17 +52,31 @@
  * }
  * }</pre>
  *
+ * <p>It is possible to annotate a class and let the internal framework encode/decode a class automatically.</p>
```

Comment:
"encode/decode a class automatically" => "encode/decode instances of that class automatically"